packages/styles/panel.css

```diff
@@ -3,6 +3,22 @@
   --panel-background-color: var(--white);
   --panel-heading-color: var(--gray-90);
   --panel-padding: var(--space-small);
+  --panel-content-padding: var(--space-small);
+  --panel-content-margin: 0 calc(var(--space-small) * -1);
+  --panel-header-margin: calc(var(--space-smaller) * -1)
+    calc(var(--space-small) * -1);
+  --panel-header-padding: var(--space-smaller) var(--space-small);
```

Comment:
Talking with @shawnsharpDQ, we can keep the top/bottom padding of the header relative to the standard padding. So this value can just be for the left/right padding and have the following in the class styles:

```css
padding: max(calc(var(--panel-header-padding, --panel-padding) - 4px), 0) var(--panel-header-padding, --panel-padding))
```

Note: The format depends on what we decide for the comment prior to this.